### PR TITLE
Robot test for onboarding phase + Jmeter probe for industrial pilot

### PR DESCRIPTION
**Pull request by sinaure**
- Robot test for onboarding phase 

- Jmeter probe for industrial pilot

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (felipevicens) [inline .project]: Add this file in the .gitignore

File: .pydevproject [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (felipevicens) [inline .pydevproject]: Add this file in the .gitignore

File: .project (hunk 1 above)
Comment (sinaure) [reply .project]: done!

File: .pydevproject (hunk 2 above)
Comment (sinaure) [reply .pydevproject]: ok!

**Pull request closed without merging**